packages/webapp/src/inductions/components/induction-journeys/third-party-journey.tsx

```diff
@@ -0,0 +1,120 @@
+                            : InductionStepGenesis.Profile
+                    }
+                >
+                    <WaitingForProfile induction={induction} />
+                </Container>
+            );
+        case InductionStatus.PendingCeremonyVideo: // not possible in Genesis mode
+            return (
+                <Container
+                    step={InductionStepInvitee.PendingVideoAndEndorsements}
+                    memberPreview={memberData}
+                >
+                    <WaitingForVideo induction={induction} />
+                </Container>
+            );
+        case InductionStatus.PendingEndorsement: // not possible in Genesis mode
+            return (
+                <Container
+                    step={InductionStepInvitee.PendingVideoAndEndorsements}
+                    memberPreview={memberData}
+                >
+                    <Heading size={1} className="mb-2">
+                        Endorsements
+                    </Heading>
+                    <InductionExpiresIn induction={induction} />
+                    <EndorsementsStatus endorsements={endorsements} />
+                    <Text>Waiting for all witnesses to endorse.</Text>
+                </Container>
+            );
+        case InductionStatus.PendingDonation:
+            return (
+                <Container
+                    step={
+                        isCommunityActive
+                            ? InductionStepInvitee.Donate
+                            : InductionStepGenesis.Donate
+                    }
+                    memberPreview={memberData}
+                >
+                    <Heading size={1} className="mb-2">
+                        Pending donation
+                    </Heading>
+                    <InductionExpiresIn induction={induction} />
+                    <EndorsementsStatus endorsements={endorsements} />
+                    {isCommunityActive ? (
+                        <Text>
+                            This induction is fully endorsed! As soon as the
+                            prospective member completes their donation to the
+                            Eden community, their membership will be activated
+                            and their Eden NFTs will be minted and distributed.
+                        </Text>
+                    ) : (
+                        <Text>
+                            As soon as this prospective member completes their
+                            donation to the Eden community, their membership is
+                            ready for activation. Once all Genesis members are
+                            fully inducted, memberships will be activated and
+                            Eden NFTs will be distributed.
+                        </Text>
+                    )}
+                </Container>
+            );
+        default:
+            return <p>Unknown error</p>;
+    }
+};
+
+export default ThirdPartyJourney;
```

Comment:
same comment about extracting the branches to its own components like suggested above for invitee and inviter journeys. although here the branches logics is very thin, not that much benefit by doing that, but still great